packages/core-tabs/core-tabs.js

```diff
@@ -52,6 +52,20 @@ addEvent(UUID, 'keydown', (event) => {
   }
 })
 
+// Use capture to support older versions of firefox
+addEvent(UUID, 'focus', (event) => {
+  if (event.target && event.target.parentElement.getAttribute('role') === 'tablist') {
+    queryAll(event.target.parentElement.children).forEach((tab) => tab.setAttribute('tabindex', '-1'))
+  }
+}, true)
```

Comment:
Too risky, and why check for `event.target`? :) we should check for `UUID`

```
addEvent(UUID, 'focus', ({target}) => { // Make sure there is no tabstops when inside tablist
  const tablist = event.target.parentElement
  if (target.getAttribute('tab') && tablist.hasAttribute(UUID)) {
    queryAll(tablist.children).forEach((tab) => tab.setAttribute('tabindex', -1))
  }
}, true)
```

Reply:
Yup! Will check for `UUID` but drop the check for `tab` as we have the requirement for tabs to be direct children.